lib/randomization_util.rb

```diff
@@ -2,10 +2,10 @@ class RandomizationHelperUtil
   def self.delta_seconds(checkouts)
     # Determine min and max dates of checkouts
     checkout_dates = checkouts.map { |checkout| checkout.created }.sort
+    return 0 if checkout_dates.empty?
```

Comment:
So I did `Time.parse 0` with the irb on my Terminal, it still looks broken. Maybe I didn't test it with a right way?

Reply:
If there are no checkouts, it shouldn't ever call `Time.parse 0`, because the `return` statement should interrupt the method execution

Reply:
ohh I see! I missed the`return`. Thanks for explaining this to me.